{"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/swiftpy\/interpretererror\/localizederror-implementations"]}],"kind":"article","schemaVersion":{"major":0,"patch":0,"minor":3},"topicSections":[{"title":"Instance Properties","identifiers":["doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/failureReason","doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/helpAnchor","doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/recoverySuggestion"],"generated":true,"anchor":"Instance-Properties"}],"sections":[],"identifier":{"url":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/LocalizedError-Implementations","interfaceLanguage":"swift"},"metadata":{"roleHeading":"API Collection","modules":[{"name":"SwiftPy"}],"role":"collectionGroup","title":"LocalizedError Implementations"},"hierarchy":{"paths":[["doc:\/\/SwiftPy\/documentation\/SwiftPy","doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError"]]},"references":{"doc://SwiftPy/documentation/SwiftPy/InterpreterError/recoverySuggestion":{"type":"topic","role":"symbol","url":"\/documentation\/swiftpy\/interpretererror\/recoverysuggestion","abstract":[],"title":"recoverySuggestion","kind":"symbol","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"recoverySuggestion","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/recoverySuggestion"},"doc://SwiftPy/documentation/SwiftPy/InterpreterError":{"identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError","title":"InterpreterError","kind":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"InterpreterError","kind":"identifier"}],"abstract":[],"type":"topic","role":"symbol","navigatorTitle":[{"text":"InterpreterError","kind":"identifier"}],"url":"\/documentation\/swiftpy\/interpretererror"},"doc://SwiftPy/documentation/SwiftPy/InterpreterError/failureReason":{"identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/failureReason","type":"topic","url":"\/documentation\/swiftpy\/interpretererror\/failurereason","role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"failureReason","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"kind":"symbol","title":"failureReason","abstract":[]},"doc://SwiftPy/documentation/SwiftPy":{"role":"collection","url":"\/documentation\/swiftpy","abstract":[{"text":"A fast and lightweight Python interpreter with Swift macro binding tools.","type":"text"}],"type":"topic","title":"SwiftPy","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy","kind":"symbol"},"doc://SwiftPy/documentation/SwiftPy/InterpreterError/helpAnchor":{"url":"\/documentation\/swiftpy\/interpretererror\/helpanchor","abstract":[],"kind":"symbol","type":"topic","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/InterpreterError\/helpAnchor","title":"helpAnchor","role":"symbol","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"helpAnchor"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}]}}}